lib/undercover/cli.rb

```diff
@@ -33,6 +34,25 @@ def self.run(args)
     end
     # rubocop:enable Metrics/MethodLength, Metrics/AbcSize
 
+    def self.build_opts(args)
+      configuration = project_options.concat(Array(args))
```

Comment:
`CLI.run` is always called with an array, so I think it's ok to just use `args`

Reply:
Oh, I ran into problems with this but it was just the way I was running the tests. Thanks, fixed.